src/entt/config/config.h

```diff
@@ -27,8 +27,12 @@
 
 
 #ifndef ENTT_ASSERT
-#   include <cassert>
-#   define ENTT_ASSERT(condition) assert(condition)
+#   ifndef ENTT_DISABLE_ASSERT
```

Comment:
I'd probably do the opposite and make disabling asserts higher priority.
After all, you redefine `ENTT_ASSERT` in code but you could want to run the application without it for once.
With your changes, you can never disable asserts if you redefined `ENTT_ASSERT` instead. Does it make sense?